src/lib/Iqons.js

```diff
@@ -0,0 +1,223 @@
+class Iqons {
+
+    /* Public API */
+
+    /**
+     * @param {string} text
+     */
+    static async svg(text) {
+        const hash = this._hash(text);
+        return this._svgTemplate(
+            parseInt(hash[0]),
+            parseInt(hash[2]),
+            parseInt(hash[3] + hash[4]),
+            parseInt(hash[5] + hash[6]),
+            parseInt(hash[7] + hash[8]),
+            parseInt(hash[9] + hash[10]),
+            parseInt(hash[11])
+        );
+    }
+
+    /**
+     * @param {string} text
+     */
+    static async toDataUrl(text) {
+        const base64string = btoa(await this.svg(text));
+        return `data:image/svg+xml;base64,${base64string.replace(/#/g, '%23')}`;
+    }
+
+    /**
```

Comment:
This can be removed in keyguard

Reply:
Placeholders are set as the image now as long as no address is set.